src/ng-select/themes/bootstrap4.theme.scss

```diff
@@ -0,0 +1,381 @@
+@import "bootstrap/scss/bootstrap";
```

Comment:
This is tightly dependent on a bootstrap package which is something that we don't have in other themes. Consider either use variables from bootstrap here without importing package or add optional dependency for that within the package.json.